index.bnd

```diff
@@ -7,4 +7,7 @@
         description="Use the maven directory layout for sources and binaries", \
     bndtools/workspace-templates/osgi; \
         name=osgi; \
-        description="OSGi R8 with Felix distribution"
\ No newline at end of file
+        description="OSGi R8 with Felix distribution", \
+    mnlipp/de.mnl.osgi/de.mnl.osgi.bnd.repository/workspace-template; \
```

Comment:
Thanks for your contribution @mnlipp 

```suggestion
    mnlipp/de.mnl.osgi/tree/b96e0a8877bad1c68cdc050d5854829253ef63bb/workspace-template; \
```

Could you please change it to this, to pin it to this specific commit SHA? (points to [here](https://github.com/mnlipp/de.mnl.osgi/tree/b96e0a8877bad1c68cdc050d5854829253ef63bb/de.mnl.osgi.bnd.repository/workspace-template))

This way we make sure that the template fragment which is selected and used by the user in Eclipse is always in a predictable state of this PR.